README.md

```diff
@@ -146,6 +149,41 @@ In the above example `results` is an object with the following structure:
     },
 }
 ```
+#### `resizeBrowserToBodyHeight`
+Is a basic command, it take one optional parameter `offset` and does what it says.
```

Comment:
It's not very clear to me what `offset` actually does. Looking at the code it appears to increase the height of the screenshot by whatever value you pass. Would that not just result in blank space at the bottom of the screenshot?

Reply:
This is correct, I've tried to improve the description of it but it is to allow for extra height in circumstances where a fixed/absolute element (say a footer) may overlay an area of the body but the desire it to capture the full body visually and the overlay beneath it, so we add the required adjustment to the height.